src/openforms/conf/base.py

```diff
@@ -287,6 +287,7 @@
                 "django.template.context_processors.request",
                 "django.contrib.auth.context_processors.auth",
                 "django.contrib.messages.context_processors.messages",
+                "openforms.forms.context_processors.language_info",
```

Comment:
seeing you only access `settings.LANGUAGES`, you can use the built-in context processor: https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#i18n-context-processor

But probably just using `{% get_available_languages as languages %}` will be sufficient